pubsub/pulsar/pulsar.go

```diff
@@ -149,6 +156,23 @@ func parsePulsarMetadata(meta pubsub.Metadata) (*pulsarMetadata, error) {
 		m.Token = val
 	}
 
+	for k, v := range meta.Properties {
+		if idx := strings.LastIndex(k, topicJSONSchemaIdentifier); idx > 0 {
```

Comment:
This would also match `bla.jsonschema.avroschema` which seems wrong.

Shouldn't the last index be exactly length of string minus length of `topicJSONSchemaIdentifier`?

Reply:
idx == len(k) - len(topicJSONSchemaIdentifier) - 1

Reply:
Done